src/ILCompiler/netcoreapp/ilc.csproj

```diff
@@ -1,4 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
+<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
This import is not going to work here - it is what is likely causing all the build breaks.

Reply:
Removed in 2nd commit.